Library/Homebrew/formulary.rb

```diff
@@ -36,8 +36,12 @@ def self.load_formula(name, path, contents, namespace)
     end
   end
 
-  def self.load_formula_from_path(name, path)
-    contents = path.open("r") { |f| set_encoding(f).read }
+  def self.formula_contents_from_path(path)
+    contents = path.open("r") { |f| set_encoding(f).read } unless contents
```

Comment:
I'm pretty 👎 on modifying `formulary` to improve the output of a `--dry-run` option just used by developers.